### PR TITLE
Jorm: add an API "geo_status" providing stats on georef data

### PR DESCRIPTION
**Pull request by kinnou02**
This PR is related to http://jira.canaltp.fr/browse/NAVITIAII-2150
The goal of this API is to provide statistics on georef data, it isn't
at destination of our users but more for the data team, it will help
them see the impacts of their update

Maybe georef_stats would be better
require: https://github.com/CanalTP/navitia-proto/pull/63

More is coming

### CODE REVIEW

File: source/jormungandr/jormungandr/interfaces/v1/GeoStatus.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            'nb_ways': fields.Raw,
+            'nb_addresses': fields.Raw,
+            'nb_poi': fields.Raw,
+        })
+}
+
+class GeoStatus(Resource):
+    @marshal_with(geo_status)
+    def get(self, region):
+        response = protobuf_to_dict(i_manager.dispatch({}, "geo_status", instance_name=region), use_enum_labels=True)
```
Comment (antoine-de) [inline source/jormungandr/jormungandr/interfaces/v1/GeoStatus.py]: why do you need this ? can't you directly marshal the protobuf ?

**Comment by antoine-de** [comment]
you have conflicts

**Comment by antoine-de** [comment]
OK for me

**Comment by antoine-de** [comment]
you need to rebase

**Comment by kinnou02** [comment]
I have rename the api in `_geo_status` since it's a private api as we can say

**Pull request merged**